geonode/base/models.py

```diff
@@ -2127,3 +2128,50 @@ class GroupGeoLimit(models.Model):
 class ExtraMetadata(models.Model):
     resource = models.ForeignKey(ResourceBase, null=False, blank=False, on_delete=models.CASCADE)
     metadata = JSONField(null=True, default=dict, blank=True)
+
+
+class AssetManager(PolymorphicManager):
+    pass
```

Comment:
What is the usage of having an empty manager?

Reply:
just a temporary placeholder